### PR TITLE
Add load indicator when getting admin repo, org, gist information.

### PR DESCRIPTION
**Pull request by MichaelTsengLZ**
Can anyone help me with the load indicator?  
I need it and it also mentioned in [issue#112](https://github.com/cla-assistant/cla-assistant/issues/112).

I choose to use angular-busy. But my current issue is that the backdrop(the white shade) cannot cover the whole background. So the 'Create New Link' button is half covered by the white backdrop. I expect the backdrop can cover the whole content background.

image

FYI, this pull request is not finished yet. And [Line 294](https://github.com/cla-assistant/cla-assistant/compare/master...Microsoft:miczeng/addIndicator?expand=1#diff-058756afa01fec41fdc06f6d452daf65R294) is my quick experiment to make the indicator last longer.

### CODE REVIEW

**Comment by KharitonOff** [comment]
We have already a loading indicator. Let us reuse it here :)
It could look like:
home.html
```
<div class="loading-indicator" ng-show="isLoading">
	<img src="/assets/images/link_inactive.svg" class="center-div spin-img ">
</div>
<!-- ----------ACTIVATE a CLA---------- -->
<div id="activated_cla" ng-show="user.value.admin" class="row content-block">
...
```
where isLoading could be set to false in your .then()

app.scss 
```
.spin-img {
  height: 60px;
  margin: -30px 0 0 -30px;
  -webkit-animation: spin 4s linear infinite;
  -moz-animation: spin 4s linear infinite;
  animation: spin 4s linear infinite;
}

.loading-indicator {
  position: fixed;
  right: 0;
  width: 100%;
  height: 100%;
  z-index: 1000;
  background: rgba(255, 255, 255, 0.8);
}

.center-div {
  margin: auto;
  position: absolute;
  top: 0;
  left: 0;
  bottom: 0;
  right: 0;
}

.center-div .spin-img {
  top: -60px
}
```
.loading-indicator, .center-div and '.center-div .spin-img' are the new classes, which should come after .spin-img

the result would look like this:
screen shot 2017-09-07 at 15 45 18

**Comment by MichaelTsengLZ** [comment]
@KharitonOff Thank you so much for your guidance. 👍

**Comment by MichaelTsengLZ** [comment]
This is the final UI:
image

File: src/client/controller/home.js [hunk 1]
```diff
@@ -290,13 +295,17 @@ module.controller('HomeCtrl', ['$rootScope', '$scope', '$document', '$HUB', '$RP
             };
 
             getUser().then(function () {
-                getOrgs().then(function () {
-                    getLinkedOrgs();
-                    getRepos();
+                $scope.isLoading = true;
+                $q.all([
```
Comment (MichaelTsengLZ) [inline src/client/controller/home.js]: I made a little small refactor here to:
1. getOrgs() is in line with getRepos() by putting getLinkedOrgs in it.
2. Use q.all() to get orgs, repos and gists parallelly.

Comment (KharitonOff) [reply src/client/controller/home.js]: That looks and works much better now 👍  Thanks!

**Pull request merged**